core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -38,14 +38,10 @@ angular.module('oppia').directive('topNavigationBar', [
       restrict: 'E',
       scope: {
         headerText: '=',
-        subheaderText: '='
+        subheaderText: '=',
+        toggle: '&'
       },
-      bindToController: {
-        backButtonShown: '<'
```

Comment:
Why removing `backButtonShown`?

Reply:
It was not used in both `top-navigation-bar.direcitve.ts` and `top-navigation-bar.directive.html`. So, removed it.